examples/Demo/Shared/Components/SiteSettingsPanel.razor

```diff
@@ -3,7 +3,7 @@
 <div>
     <FluentDesignTheme @bind-Mode="@Mode"
                        @bind-OfficeColor="@OfficeColor"
-                       Direction="@(Direction ? "ltr" : "rtl")"
+                       Direction="@(Direction ? LocalizationDirection.ltr : LocalizationDirection.rtl)"
```

Comment:
Following HTML doc, Direction can be `ltr`, `rtl` or `auto` (and maybe empty?)
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir

And I think we need to refactor the enum using Pascal case.
```csharp
public enum LocalizationDirection
{
    Ltr,
    Rtl,
    Auto
}
```

Reply:
Agree that is is part of the standard but we do not use 'auto' in any of our components (atm). How would we handle something like if 'auto' would be a valid option? 
```
protected string? StyleValue => new StyleBuilder(Style)
    .AddStyle("left", $"{HorizontalPosition!.Value.ToString(CultureInfo.InvariantCulture)}%", () => HorizontalPosition.HasValue && GlobalState.Dir == LocalizationDirection.Ltr)
    .AddStyle("right", $"{HorizontalPosition!.Value.ToString(CultureInfo.InvariantCulture)}%", () => HorizontalPosition.HasValue && GlobalState.Dir == LocalizationDirection.Rtl)
```

I'll do the Pascal refactoring

Reply:
I think `Auto` should be present in the Enum and that you should apply a "default value" for some components that need LTR or RTL. But you can't dictate the direction of the entire website.

Reply:
Added new LeftToRight an RightToLeft values and obsoleted th 'old' values